### ISSUE TITLE
Module parse failed 

### ISSUE THREAD

**Issue by kajm** [react-boilerplate/react-boilerplate#1595]
Hi guys!

When I add a styles.css file in one of my components I get this "module parse failed" - error and its the message redirect me to the new file that I have made. I have tried googled it a couple of times, but I just can't figure it out what the problem is. 

I have currently this versions installed:
React-boilerplate: 3.4.0
NPM: 4.1.2
Node: 7.5.0
Google Chrome

and here is the error:
<img width="1180" alt="skarmavbild 2017-02-16 kl 20 03 44" src="https://cloud.githubusercontent.com/assets/9610681/23036474/40ca1d2c-f483-11e6-955d-f9bd4c5edeb1.png">

What could be wrong?

best regards,
Karl

**Comment by gggdomi** [comment]
Hey @kajm,
I had the exact same problem, and I think I finally figured out the issue. It lies in `webpack.base.babel.js` when configuring CSS loaders:
```javascript
...
{
      // Do not transform vendor's CSS with CSS-modules
      // The point is that they remain in global scope.
      // Since we require these CSS files in our JS or CSS files,
      // they will be a part of our compilation either way.
      // So, no need for ExtractTextPlugin here.
      test: /\.css$/,
      include: /node_modules/,
      loaders: ['style-loader', 'css-loader'],
}
...
```

Due to  `include: /node_modules/`, only the .css files in `node_modules` will be processed by the loaders.
This new behaviour has been introduced by #1100 when switching to styled-components.
To import my local CSS file, I added the old code just before the previous rule:

```javascript
...

    }, {
      // Transform our own .css files with PostCSS and CSS-modules
      test: /\.css$/,
      exclude: /node_modules/,
      loaders: ['style-loader', 'css-loader'],
      // BTW, if you want to enable CSS modules you can use this instead:
      // loaders: 'style-loader!css-loader?modules=true&localIdentName=[name]__[local]___[hash:base64:5]',
    }, {
      // Do not transform vendor's CSS with CSS-modules
      // The point is that they remain in global scope.
...
```

Not sure if it's a good idea (I'm new to React and this boilerplate), but at least the error is gone :)

**Comment by samit4me** [comment]
This is definitely an issue. Not sure how this was missed but this boilerplate has been using webpack 2 for quite a long time and it looks like the loaders need updating as @kajm mentioned. See https://webpack.js.org/guides/migrating/#module-loaders-is-now-module-rules

**Comment by kajm** [comment]
@gggdomi thanks you for your answer, I will try your solution and see if it works for me too.

**Comment by KarandikarMihir** [comment]
Closing this. Resolved in #1599 Cheers! :beers:
